core/src/main/java/greencity/annotations/RatingCalculationEnum.java

```diff
@@ -0,0 +1,21 @@
+package greencity.annotations;
+
+import lombok.Getter;
+
+@Getter
+public enum RatingCalculationEnum {
```

Comment:
There is no need to add `Enum` to the name, the same way as we don't add `Class` to the class name or `I` to interface name.